python/singa/tensor.py

```diff
@@ -107,9 +107,14 @@ def __init__(self, shape=(), device=None, dtype=float32,
         self.dtype = self.data.data_type()
         self.requires_grad = requires_grad
         self.stores_grad = stores_grad
+        if name is None:
+            self.name = 'Dummy#{}'.format(Tensor.tensor_count)
```

Comment:
we only need to give a name to the input tensors.
for the tensors generated by operations, the name should be set by the output_name() function.

Reply:
`node.append(helper.make_node('Concat',
                                         inputs=inputs,
                                         outputs=outputs,
                                         name=op.name,
                                         axis=op.axis))`
for the tensors generated by operations
output parameter is set by output_name() 
name parameter is set by op.name
should i change name parameter to output_name() too? same as output parameter?